### DISCUSSION TITLE
Community Discord Server For The Bot!!!

### DISCUSSION THREAD

**Discussion by nehcap** [tanjeffreyz/auto-maple#69] [General]
https://discord.gg/hKfdg9xecd
https://discord.gg/hKfdg9xecd
https://discord.gg/hKfdg9xecd
https://discord.gg/hKfdg9xecd

**Comment by AMDEEEZ** [comment]
Link expired can u renew?

**Reply by zioarca96** [reply]
Did you get the new link?

**Comment by n20balloon** [comment]
Hello I'm willingly to make a server where we discuss strategies and ways to improve for this amazing bot.
anybody interested pm me.

**Reply by zioarca96** [reply]
Ehi, would you please add me through discord? Nik#8124
Thank you :))

**Reply by Mats52** [reply]
This link is out of date. Is there a new Discord link

**Comment by JohnCongroo** [comment]
I'd also like an invite to any discord for this project, thank you.

**Reply by noobishh** [reply]
hi i am interested in discussing the macro with you. you have discord?

**Comment by Mats52** [comment]
The link has expired, can you provide a new link?

**Comment by noobishh** [comment]
yes please. someone post a new link for this discord discussion! Thank you!

**Comment by Yalian** [comment]
Bump for new link

**Comment by Kailexx3** [comment]
bump for new link
